tests/Mapper/ProgrammesDbToDomain/ProgrammeMapperTest.php

```diff
@@ -17,6 +18,13 @@ class ProgrammeMapperTest extends BaseProgrammeMapperTestCase
 {
     public function testGetDomainModelBrand()
     {
+        $options = ['programmeOptions'];
```

Comment:
This test file doesn't test the mappers of any of the other models that require mappers (e.g.Image, Masterbrand) so there isn't a need to do that for options either.

In the tests in this file assume the options are blank - the value of `[]` and don't bother writing any expectations on the mockOptionsMapper. 

Testing that the Programme passes things through to the OptionMapper is done in the ProgrammeMapperOptionsMappingTest